CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Enhancements:
 Deprecations:
  - Removed support for the .NET Framework < 4.5 and .NET Standard 1.x. (@stakx, #495, #496)
  - Removed support for Code Access Security (CAS). (@stakx, #502)
+ - Minimized support for Remoting. This library no longer defines any types deriving from `MarshalByRefObject`, and `ProxyUtil.IsProxy` (which used to recognize remoting/"transparent" proxies) now tests only for DynamicProxy proxies. (@stakx, #507)
```

Comment:
"Minimized support for Remoting" sounds a bit strange, we've really dropped support for .NET remoting, without MBRO you can't make those RPCs across AppDomains.

Reply:
True, I suppose. I'll replace _"Minimized"_ with _"Removed"_. I'll probably keep the 2nd sentence, seems like a short explanation what exactly "removal" means *might* be mildly helpful.